pages/koneet-kiertoon.tsx

```diff
@@ -0,0 +1,411 @@
+import Head from 'next/head'
+import styled from 'styled-components'
+import { Content } from '../components/Content/Content'
+import { H1 } from '../components/Title/Title'
+import { Footer } from '../components/Footer/Footer'
+import { GetServerSideProps, InferGetServerSidePropsType } from 'next'
+import { Gallery } from '../components/Gallery/Gallery'
+import { InfoBox } from '../components/InfoBox/InfoBox'
+
+const Section = styled.div`
+
+  margin-top: 3rem;
+  margin-bottom: 1rem;
+
+  h2 {
```

Comment:
Personal opinion, but I don't like putting these child components inside styled-components' CSS. Instead, I create a new component for them: `const SectionTitle = styled.h2...`. It makes the code more maintainable and readable.

Reply:
This change can be implemented later. Feel free to make a PR after this one is merged.